polkadot/zombienet_tests/functional/0015-coretime-shared-core.zndsl

```diff
@@ -0,0 +1,16 @@
+Description: CT shared core test
+Network: ./0015-coretime-shared-core.toml
+Creds: config
+
+validator: reports node_roles is 4
+
+# register paras 2 by 2 to speed up the test. registering all at once will exceed the weight limit.
+validator-0: js-script ./0015-force-register-paras.js with "2000,2001" return is 0 within 600 seconds
+validator-0: js-script ./0015-force-register-paras.js with "2002,2003" return is 0 within 600 seconds
+# assign core 0 to be shared by all paras.
+validator-0: js-script ./assign-core.js with "0,2000,14400,2001,14400,2002,14400,2003,14400" return is 0 within 600 seconds
+
+collator-2000: reports block height is at least 6 within 200 seconds
```

Comment:
With this high timeouts and low block heights, aren't we at a high-risk of the test working accidentally.

Reply:
each parachain should get one block every 4 relay chain blocks. so one in 24 seconds (which would amount to about 144 seconds). An extra minute is needed for the session change (we're registering the parachains manually, because zombienet would otherwise also assign cores)

Reply:
Alright makes, sense then the timeouts aren't that high. Thank you!